pkg/apis/kops/cluster.go

```diff
@@ -241,6 +241,11 @@ type ClusterSpec struct {
 
 	// Tags for AWS instance groups
 	CloudLabels map[string]string `json:"cloudLabels,omitempty"`
+
+	// vSphere required specs
+	VSphereServer       string
```

Comment:
The CloudLabels above these seems is a AWS specific things... So I suppose it's ok to add VC specific things?